src/bpftrace.h

```diff
@@ -182,7 +182,10 @@ class BPFtrace {
   BpfBytecode bytecode_;
   StructManager structs;
   std::map<std::string, std::string> macros_;
-  std::map<std::string, uint64_t> enums_;
+  // Map of enum variant_name -> (variant_value, enum_name)
```

Comment:
super nit (just language consistency)
```suggestion
  // Map of enum variant_name to (variant_value, enum_name)
```